frontend/src/App.js

```diff
@@ -26,7 +27,8 @@ function App() {
             <Route path="/profile/edit" element={<EditProfile />} />
             <Route path="/profile/follows" element={<Follows />} />
             <Route path="/admin/categories" element={<AdminCategories />} />
-            <Route path="/admin/category/edit" element={<EditCategory />} />
+            <Route path="/admin/category/edit/:id" element={<EditCategory />} />
```

Comment:
better way to do in routing `/key/action`
```suggestion
            <Route path="/admin/category/:id/edit" element={<EditCategory />} />
```